rules/windows/process_creation/proc_creation_win_lolbin_syncappvpublishingserver_vbs_execute_psh.yml

```diff
@@ -19,7 +19,7 @@ detection:
     selection:
         CommandLine|contains|all:
             - '\SyncAppvPublishingServer.vbs'
-            - '"\n;'
+            - ';'  # at a minimum, a semi-colon is required
```

Comment:
A single character can be problematic for some backends

Reply:
Well, at least it requires both strings

Reply:
I thought regex was overkill for it so I decided on this.